features/distro/src/main/feature/feature.xml

```diff
@@ -9,4 +9,8 @@
         <bundle>mvn:org.openhab.ui.bundles/org.openhab.ui.iconset.classic/${project.version}</bundle>
     </feature>
 
+    <feature name="openhab-addon-discovery" description="Discovery Information" version="${project.version}">
+        <configfile finalname="${openhab.userdata}/addons/openhab-addons.xml" override="false">mvn:org.openhab.addons.features.karaf/org.openhab.addons.features.karaf.openhab-addons-external/${project.version}/xml/addons</configfile>
```

Comment:
Just checking: does the source file name here, match the source file name in https://github.com/openhab/openhab-addons/blob/a9a79b09bf5473bcdd08717f043f8ddfe48fbc6a/features/openhab-addons-external/pom.xml#L35C1-L35C1 ??

Reply:
Correct.